### ISSUE TITLE
Implement Standard Scaler to Feed All Models

### ISSUE THREAD

**Issue by rileymccarty** [rileymccarty/Average_Joes_Ancient_Artifacts#51]
Ensure that we are feeding all models with Scaled Data.

Scale data based on Training set so there is not leakage from test set.
